rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -7970,3 +7971,98 @@ fn subnet_info_ingress_fails() {
             "cannot be called by a user",
         );
 }
+
+#[test]
+fn node_metrics_history_update_succeeds() {
```

Comment:
I suppose this should be `node_metrics_history_ingress_update_fails` since it's doing an ingress update call.